src/setup/serverless-config.go

```diff
@@ -269,14 +365,28 @@ func RemoveCloudflareAllWorkers(subExperiments []SubExperiment) []string {
 	return removeServiceMessages
 }
 
-// Removes a single Cloudflare Worker specified by name
+// RemoveCloudflareSingleWorker removes a single Cloudflare Worker specified by name
 func RemoveCloudflareSingleWorker(workerName string) string {
 	log.Infof("Removing Cloudflare Worker %s...", workerName)
 	removeWorkerCommand := exec.Command("wrangler", "delete", "--name", workerName, "--force")
 	removeMessage := util.RunCommandAndLog(removeWorkerCommand)
 	return removeMessage
 }
 
+// RemoveAlibabaAllServices removes all Azure services
+func RemoveAlibabaAllServices(path string, numSubExperiments int) []string {
+	nameOfBucketToDelete := fmt.Sprintf("oss://sls-%s-%s", ALIBABA_DEFAULT_ACCOUT_ID, ALIBABA_DEFAULT_REGION)
```

Comment:
Typo, same as above.

Also, if users want to use their own Alibaba account ID, how will their ID be known and handled? It looks like `RemoveAlibabaAllServices` is hardcoded to only use `ALIBABA_DEFAULT_ACCOUT_ID` at the moment. 

Perhaps this value should be read from the env instead and only default when none is specified? I see that it is defined as `ALIYUN_ACCOUNT_ID` in the workflow file.

Reply:
Good suggestion, thanks! I've changed it to read the value of the environment variable and fallback to the default account ID